src/core/unit_tests/CMakeLists.txt

```diff
@@ -4,45 +4,93 @@
 
 # This file is part of ESPResSo.
 
-# ESPResSo is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
+# ESPResSo is free software: you can redistribute it and/or modify it under the
```

Comment:
This formatting should not happen; please revert this change and add the missing `#` symbol in empty lines of the license header.